sale_report_salesman/views/menu_views.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="sale.menu_sale_report" model="ir.ui.menu">
```

Comment:
What will happen when the module is uninstalled? :wink: 
It may be necessary to use uninstall_hook to remove the group from this menu.

Reply:
New knowledge achieved. Thanks! I just added :)